spi/core-spi/build.gradle.kts

```diff
@@ -25,6 +25,8 @@ dependencies {
     api("com.fasterxml.jackson.core:jackson-annotations:${jacksonVersion}")
     api("com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}")
     api("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:${jacksonVersion}")
+    api("io.opentelemetry:opentelemetry-api:${openTelemetryVersion}")
```

Comment:
this can be converted to an `implementation` dependency, if the `Telemetry` class provides a parameterless constructor, which could internally initialize with `noop()`:
```java
public Telemetry() {
    this.openTelemetry = OpenTelemetry.noop();
}
```

Reply:
Thanks for the hint. I needed to add a second "implementation" dependency in the core/boot module for it to work as the `ExtensionLoader` still needs access to the open telemetry API.